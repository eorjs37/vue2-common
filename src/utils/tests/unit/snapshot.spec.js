import repeat from '@/utils/snapshot'
describe('snapshot unit test', () => {
  test('repeats words three times', () => {
    expect(repeat('Test', 3)).toMatchSnapshot()
  })
})
