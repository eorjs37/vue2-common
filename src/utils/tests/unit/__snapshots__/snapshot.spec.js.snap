// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot unit test repeats words three times 1`] = `"Test,Test,Test"`;
